Sources/BraintreeCoreSwift/BTAPIHTTP.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+// TODO: Make internal once rest of core is in Swift
+@objcMembers public class BTAPIHTTP: BTHTTP {
+
+    let accessToken: String?
+
+    @objc(initWithBaseURL:accessToken:)
+    public init(url: URL, accessToken: String? = "") {
+        self.accessToken = accessToken
+        super.init(url: url)
```

Comment:
Making the `accessToken` non-optional would also let us remove the `BTHTTP.init(url: URL)` internal helper method

Reply:
We would still need to call one of the super inits here which would mean we would need a client token, tokenization key, or an authorization fingerprint. I don't think we'd have any of these from the access token correct?

Reply:
Yeh - I think my comment gets into the territory of rearchitecting our network layer and beyond converting to Swift. We can disregard for now!

Reply:
Ah, gotcha. Yeah we should certainly look at that more holistically at some point - would be great to modernize our networking layer but agree that is a large task that touches a lot of pieces. 🚀